src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -166,26 +166,60 @@
 import org.apache.commons.io.FilenameUtils;
 import org.apache.commons.io.IOUtils;
 
+/**
+ * This class contains various hooks and utility methods for Minecraft Forge.
+ */
 public class ForgeHooks
```

Comment:
should ForgeHooks provide java doc?

Reply:
Probably, I just added some.

Reply:
I'd prefer removing javadocs from here, this class is used too much to have generic javadocs.

Reply:
Removed